tests/unit/helpers/eventTimeStamping.test.js

```diff
@@ -124,7 +125,7 @@ describe('addTimeStamp', () => {
   it('should add startDate timestamp and updateEvent', async () => {
     const event = { _id: new ObjectID() };
     const startDate = new Date();
-    const payload = { action: 'manual_timestamping', reason: 'qrcode', startDate };
+    const payload = { action: { $in: TIME_STAMPING_ACTIONS }, reason: 'qrcode', startDate };
```

Comment:
il faut juste remplacer  `action: 'manual_timestamping'` par `action: 'manual_time_stamping'`. La fonction va créer un EventHistory avec cette action, ça ne peut donc par être une query `{ $in: TIME_STAMPING_ACTIONS }` 😄

Reply:
Même chose pour les autres

Reply:
Ah mais oui, j'avais pas compris pardon! Je rectifie